backend/src/apps/donations/views/donation_api_view.py

```diff
@@ -6,12 +6,6 @@
 
 
 class DonationAPIView(APIView):
-    @staticmethod
-    def get(request) -> JsonResponse:
-        donations = Donation.objects.all()
-        serializer = DonationsSerializer(donations, many=True)
-        return JsonResponse(data=serializer.data, safe=False)
-
     @staticmethod
     def get_by_id(request, pk: int) -> JsonResponse:
```

Comment:
Implement `get` method instead because the incoming request is dispatched to an appropriate handler method such as .get() or .post()